src/test/java/com/navfort/step_definitions/US_163_StepDefinitions.java

```diff
@@ -0,0 +1,106 @@
+package com.navfort.step_definitions;
+
+import com.navfort.pages.LoginPage;
+import com.navfort.pages.QuickLaunchpadPage;
+import com.navfort.pages.VehiclesPage;
+import com.navfort.utilities.Driver;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
+import com.navfort.utilities.BrowserUtils;
+
+import java.util.List;
+
+public class US_163_StepDefinitions {
+
+    QuickLaunchpadPage quickLaunchpadPage = new QuickLaunchpadPage();
+    VehiclesPage vehiclesPage = new VehiclesPage();
+
+    @Given("the user is on the vehicle page AS")
+    public void the_user_is_on_the_vehicle_page_as() {
+        BrowserUtils.waitForPageToLoad(5);
+        BrowserUtils.waitForClickablility(quickLaunchpadPage.FleetDropdown, 5);
+        Actions action = new Actions(Driver.getDriver());
+        action.moveToElement(quickLaunchpadPage.FleetDropdown).perform();
+        quickLaunchpadPage.VehiclesPageLink.click();
+    }
+
+    @When("user sees all the checkboxes in the table AS")
+    public void user_sees_all_the_checkboxes_in_the_table() {
+        BrowserUtils.waitForPageToLoad(5);
+        List<WebElement> checkboxesAS = Driver.getDriver().findElements(By.xpath("//*[@data-role='select-row-cell']"));
```

Comment:
Andrew, looks good. But I noticed that you located some of your WE inside of StepDefs file. I think it's better  to do in your VehiclesPage to follow POM design pattern